vb-be/src/modules/auth/auth.service.ts

```diff
@@ -85,12 +84,26 @@ export class AuthService {
 
   async refreshTokens(
     accessToken: string,
-    userInfo: UserType,
+    user: ITokenUser,
```

Comment:
passing in `UserType` requires a bunch of unnecessary data here.  We only want the email, and perhaps the `id`, once we re-implement that

Reply:
yeah thats okay no problem with that